src/routes/profil/+page.server.ts

```diff
@@ -12,7 +12,219 @@ export const load = (async ({ params }) => {
 }) satisfies PageServerLoad;
 
 export const actions = {
-	default: async ({ request, cookies }) => {
+	postExperience: async ({ request, cookies }) => {
+		const session_cookie = cookies.get('supabase-auth-token');
+
+		if (!session_cookie) {
+			throw error(500, {
+				code: 500,
+				message: 'No session cookie found',
+			});
+		}
+
+		const session = await supabase.auth
+			.setSession({
+				access_token: JSON.parse(session_cookie)[0],
+				refresh_token: JSON.parse(session_cookie)[1],
+			})
+			.catch((err) => {
+				throw error(500, {
+					code: 500,
+					message: err.message,
+				});
+			});
+
+		const user_id = session.data.user?.id;
+		const { data, error: errr } = await supabase
```

Comment:
```suggestion
		const { data, error: err } = await supabase
```

3 `r` c'est peut-être beaucoup, peut-être plutôt `err_profile`